### PR TITLE
FIX: Execution Issues with two Lectures

### PR DESCRIPTION
**Pull request by mmcky**

### CODE REVIEW

**Comment by mmcky** [comment]
@AakashGfude any ideas on why this mathjax2 config is not working to render the output. 

This `_config.yml` file specifies mathjax2 with mathjax2 config but is not longer working. 

We can either:

1. Update to mathjax3 (should be done at some point)
2. fix the `_config` pass through. 

Can you look at the `html` and let me know if mathjax2 is passed through correctly?

File: lectures/_config.yml [hunk 1]
```diff
@@ -32,7 +32,7 @@ sphinx:
         binderhub_url: https://mybinder.org
       google_analytics_id: UA-54984338-5
       persistent_sidebar: true
-    mathjax_config:
+    mathjax3_config:
```
Comment (mmcky) [inline lectures/_config.yml:35]: hey @AakashGfude if you change this to mathjax 3 we will need to update all the option keys.

For example `TeX` -> `tex` and `Macros` -> `macros`

Comment (AakashGfude) [reply lectures/_config.yml:35]: That's right @mmcky, I was checking first if the v3 works in preview. It seems to be working, I will do the necessary updates now.

Comment (mmcky) [reply lectures/_config.yml:35]: thanks @AakashGfude

Comment (AakashGfude) [reply lectures/_config.yml:35]: @mmcky the math should be okay now. Will look at the preview for a few pages.

**Comment by mmcky** [comment]
thanks @AakashGfude. @doctor-phil over to you now. Would you mind checking the preview when it is ready? @AakashGfude has upgraded the settings to use the later version of mathjax which is `v3`

**Comment by mmcky** [comment]
@AakashGfude is this going to cause an incompatibility with any `plotly` diagrams. From memory that was the main reason we used `v2` on a couple of projects.

**Comment by AakashGfude** [comment]
@mmcky Should be okay as long as the plotly is above v5.7.0. I will pin plotly version here.

**Comment by doctor-phil** [comment]
Math looks to be rendering right to me. Thanks @AakashGfude @mmcky

**Pull request merged**